src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -167,6 +168,7 @@ interfaces:
     page_streaming:
       request:
         token_field: page_token
+        page_size_field: page_size
```

Comment:
The `ListStrings` request also has a `page_size` field, but I intended to remove that one for tests where no `page_size` is specified.
Not super important, but if you remove the field from the proto then the yaml should match.